docs/gen-ai/gen-ai-output-messages.json

```diff
@@ -0,0 +1,154 @@
+{
+    "$defs": {
+        "FinishReason": {
+            "description": "Describes the reason for finishing the generation.",
+            "enum": [
+                "stop",
+                "length",
+                "content_filter",
+                "tool_call",
+                "error"
+            ],
+            "type": "string"
+        },
+        "OutputMessage": {
+            "additionalProperties": true,
+            "description": "Describes generated output.",
+            "properties": {
+                "role": {
+                    "type": "string"
+                },
+                "parts": {
+                    "items": {
+                        "discriminator": {
+                            "mapping": {
+                                "text": "#/$defs/TextPart",
+                                "tool_call": "#/$defs/ToolCallPart",
+                                "tool_call_response": "#/$defs/ToolCallResponsePart"
+                            },
+                            "propertyName": "type"
+                        },
+                        "oneOf": [
+                            {
+                                "$ref": "#/$defs/TextPart"
+                            },
+                            {
+                                "$ref": "#/$defs/ToolCallPart"
+                            },
+                            {
+                                "$ref": "#/$defs/ToolCallResponsePart"
+                            }
+                        ]
+                    },
+                    "type": "array"
+                },
+                "finish_reason": {
```

Comment:
do we need to have it given  it's already on the span? 

some ideas:
- contents are already verbose, have it duplicated is not a big problem
- we don't really need to capture 'success' (stop), we can only capture when content is not complete/filtered/etc

Reply:
let's address it as a part of https://github.com/open-telemetry/semantic-conventions/issues/1277, this is not related to this PR